### ISSUE TITLE
boolean getters "isWhatever()" not supported

### ISSUE THREAD

**Issue by Lessy** [duesenklipper/wicket-safemodel#2]
Hi,

Thank you very much for this great piece of work :-)

if you put this on line 85 in SafeModel.java then isXXX will work as well:

<pre>
            } else if (methodName.startsWith("get") || methodName.startsWith("is")) {
                int pos = methodName.startsWith("get") ? 3 : 2;
                final String propertyName = new StringBuilder().append(Character.toLowerCase(methodName.charAt(pos)))
                        .append(methodName.substring(pos + 1)).toString();
</pre>

Regards Hans

**Comment by duesenklipper** [comment]
You are correct. I'll apply this patch later. However, SafeModel is pretty much superseded by Sven Meier's LazyModel. It's inspired by SafeModel's API (nearly identical) but it is much better optimized for performance. Have a look at it here: https://github.com/wicketstuff/core/wiki/LazyModel

Thank you for the bug report though, I'll fix it!

**Comment by Lessy** [comment]
thanks for the quick answer.
Does that mean that you are using LazyModel yourself now instead of SafeModel as well?
